### ISSUE TITLE
Tech C38 example not working

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#679]
At: https://www.w3.org/WAI/WCAG21/Techniques/css/C38 the Working example: Using Adjustable Labels and Inputs for Reflow: https://www.w3.org/WAI/WCAG21/working-examples/css-fitting-inputs/index.html is a 404 DOCUMENT NOT FOUND

**Comment by alastc** [comment]
Hi Jake,

Is this the right one?
https://www.w3.org/WAI/WCAG21/working-examples/css-fit-labels-inputs/ 

-Alastair

**Comment by jake-abma** [comment]
@alastc yes, it is

**Comment by jake-abma** [comment]
I see it lacks `<meta name="viewport" content="width=device-width, initial-scale=1.0">`

**Comment by patrickhlauke** [comment]
made a PR, @jake-abma https://github.com/w3c/wcag/pull/786
